test/cpro/js/ui5/mobx/MobxModelContextBinding.test.ts

```diff
@@ -0,0 +1,48 @@
+import { MobxContextBinding } from "cpro/js/ui5/mobx/MobxContextBinding";
+import { MobxModel } from "cpro/js/ui5/mobx/MobxModel";
+import { observable } from "mobx";
+import Context from "sap/ui/model/Context";
+
+import { TestState, createTestData } from "./test-infra/TestHelper";
+
+describe("MobxModel Tests: Context Binding", () => {
+  let state: TestState;
+  let model: MobxModel<TestState>;
+  let binding: MobxContextBinding;
+  let ctx: Context;
+
+  const NEW_VALUE = "one to bind them all";
+
+  beforeEach(() => {
+    state = observable(createTestData());
+    model = new MobxModel(state);
+    ctx = new Context(model, "/complex");
```

Comment:
I've trouble to imagine how this works in a UI5 component. I have a parent component which performs the context binding (`binding="{/complex}"`). And then comes the child complonent performing its binding (`binding="{a}").

Are you sure that `model.bindContext` is the only interaction with the model?

Reply:
The way I thought of it was, that it is more a combination of both context and property/aggr binding when working with UI5 controls as a context will be bound to the parent control, the child-controls have relative bindings which will be resolved against the bound context.

I didn't think about both, parent and child component/control using only the `binding` property. I went through a few context binding scenarios and mostly observed the `[bindContext](https://github.com/SAP/openui5/blob/edb94530aa9d45ec61f79867e04eb3e7d8b529b4/src/sap.ui.core/src/sap/ui/model/ManagedObjectBindingSupport.js#L74)` being called here.

But as already written by you, that's certainly no 100% guarantee.

> And then comes the child component performing its binding (`binding="{a}").

Would that imply the child component (in this specific scenario against the test data) would have some property/aggr binding somewhere with `""` or `"/"` which resolved to the value of 'a'?